.github/workflows/lint-build-commits.yaml

```
@@ -103,7 +103,7 @@ jobs:
             ${{ github.event.pull_request.commits_url }})
           PR_FIRST_SHA=$(echo "$PR_COMMITS_API_JSON" | jq -r ".[0].sha")
           PR_PARENT_SHA=$(git rev-parse "${PR_FIRST_SHA}^")
-          git rebase --exec "make -C test build -j $(nproc)" $PR_PARENT_SHA
+          git rebase --exec "make -C test build -j $(nproc);make -C test build-darwin" $PR_PARENT_SHA
```

I think that if the first build fails, then we should cancel the second?

```suggestion
          git rebase --exec "make -C test build -j $(nproc) && make -C test build-darwin" $PR_PARENT_SHA
```